commerce-theme-minium/commerce-theme-minium/src/css/minium/components/bom.scss

```diff
@@ -314,4 +292,42 @@
 		position: relative;
 		transition: height 0.35s ease;
 	}
-}
\ No newline at end of file
+}
+
+.bom-card {
+	border-radius: 8px;
+	box-shadow: 0 0 0 0 rgba(0, 0, 0, 0);
+	overflow: hidden;
+	transition: box-shadow 0.1s ease-in, transform 0.1s ease-in;
+	
+	&.card {
+		box-shadow: 0 1px 3px -1px rgba(0, 0, 0, 0);
+
+		&:hover {
+			box-shadow: 0 10px 22px -6px rgba(0, 0, 0, 0.2);
+			transform: translateY(-4px);
+		}
+	}
+	
+	.card-body {
+		padding: 20px;
+	}
+
+	&.image-card .bom-card__folder-picture-container{
+		background-color: #E7EEFE;
+		background-image: none;
+	}
+
+	.card-title {
```

Comment:
Hey Fabio, these selectors should be sorted (assuming that doesn't break the cascade).

Reply:
@FabioDiegoMastrorilli it looks like this still needs to be sorted -- can you send a follow-up PR?